src/Microsoft.Health.Fhir.Api.UnitTests/Features/Filters/ValidateContentTypeFilterAttributeTests.cs

```diff
@@ -130,6 +130,23 @@ public async Task GivenARequestWithNotAllowedContentTypeHeader_WhenValidatingThe
             await Assert.ThrowsAsync<UnsupportedMediaTypeException>(async () => await filter.OnActionExecutionAsync(context, actionExecutedDelegate));
         }
 
+        [Theory]
+        [InlineData("\"application/fhir+json")]
+        [InlineData("]application/fhir+json")]
+        [InlineData("application\blah")]
```

Comment:
If the test logic is the same, perhaps just add these new cases to the test above. Maybe the test method name could begin with something like `GivenARequestWithUnsupportedContentTypeHeader`

Reply:
Sure. I will add the new  cases to already existing test. Thank you.